experimenter/experimenter/features/management/commands/load_feature_configs.py

```diff
@@ -170,6 +171,7 @@ def handle(self, *args, **options):
                 schema = NimbusVersionedSchema(
                     feature_config=feature_config,
                     version=feature_version,
+                    allow_coenrollment=allow_coenrollment,
```

Comment:
You are not checking for the case where the schema lands without `allowCoenrollment` but then gets it later on.